python/src/GeneralLinearModelAlgorithm_doc.i.in

```diff
@@ -355,7 +355,25 @@ optimizeParameters : bool
 Parameters
 ----------
 bounds : :class:`~openturns.Interval`
-    Bounds for covariance model parameter optimization."
+    Bounds for covariance model parameter optimization.
+
+Notes
+-----
+Parameters involved by this method are:
+
+ - Scale parameters,
+ - Amplitude parameters if output dimension is greater than one or
+   analytical sigma disabled,
+ - Additional parameters.
+
+Default bounds in the library are :math:`[100]^d` with `d` the size
```

Comment:
Is 100 hard-coded or given by a key in ResourceMap? It should be mentioned here

Reply:
It's in the RM, see GLM::setCovarianceModel